tests/test_driver.py

```diff
@@ -25,85 +26,58 @@
 RED = "\033[31m"
 
 
-def makeenv(script_path, home, test_helper_path):
-    xdg_config = home + "/xdg_config_home"
-    func_dir = xdg_config + "/fish/functions"
+def makeenv(script_path: Path, home: Path, test_helper_path: Path) -> dict[str, str]:
```

Comment:
I'd probably have put the commit that changes `str -> Path` first,
because that one is unlikely to see further change but may cause
trivial merge conflicts when squashing fixups